package.json

```diff
@@ -76,6 +76,9 @@
     "worker-loader": "^2.0.0",
     "xxhashjs": "^0.2.1"
   },
+  "resolutions": {
```

Comment:
this is only to show integration with my `enhanced-resolve` change - will remove this (& yarn.lock change) if/when pr is approved.